packages/react-storage/src/components/StorageBrowser/views/Controls/Table.tsx

```diff
@@ -127,20 +128,35 @@ export type RenderRowItem<T> = (row: T, index: number) => JSX.Element;
 interface TableControlProps<T> {
   data: T[];
   columns: Column<T>[];
+  handleDroppedFiles?: (files: File[]) => void;
```

Comment:
For future reference, event callbacks passed as props should align to the `onSomeEvent` syntax, in this case would be something like `onDrop`. No need to change this, will be updated in the comtext/UI layer refactoring

Reply:
Ah, thanks for that information. I'll leave it as-is for now, but I'll keep that in mind moving forward